wgsl/index.bs

```diff
@@ -1037,64 +1061,103 @@ Two array types are the same if and only if all of the following are true:
         even if one is signed and the other is unsigned.
         (Signed and unsigned values are comparable in this case because element counts must
         be greater than zero.)
-
-Issue: Array types should differ if they have different element strides.
-See https://github.com/gpuweb/gpuweb/issues/1534
+* They have the same [=element stride=].
 
 <div class='example fixed-size array types' heading='Example fixed-size array types'>
-  <xmp>
-    // array<f32,8> and array<i32,8> are different types:
+  <xmp highlight='rust'>
+    // array<f32,5> and array<i32,5> are different types because they have
     // different element types
-    var<private> a: array<f32,8>;
-    var<private> b: array<i32,8>;
-    var<private> c: array<i32,8u>;  // array<i32,8> and array<i32,8u> are the same type
+    var<private> a: array<f32,5>;
+    var<private> b: array<i32,5>;
+    var<private> c: array<i32,5u>;  // array<i32,5> and array<i32,5u> are the same type
 
-    let width = 8;
-    let height = 8;
+    let width = 5;
+    let height = 5;
 
-    // array<i32,8>, array<i32,8u>, and array<i32,width> are the same type.
-    // Their element counts evaluate to 8.
+    // array<i32,5>, array<i32,5u>, and array<i32,width> are the same type.
+    // Their element counts evaluate to 5.
     var<private> d: array<i32,width>;
 
     // array<i32,height> and array<i32,width> are the same type.
     var<private> e: array<i32,width>;
     var<private> f: array<i32,height>;
+
+    // array<i32,5,4> and array<i32,5> are the same type because the implicit stride of
+    // array<i32,5> is 4.
+    var<private> g: array<i32,5,4>;
```

Comment:
unfortunately, this doesn't read well :(

Reply:
How about:
> // Both the g and h arrays instantiate the same type because h's implicit stride is the same as g's explicit stride.
> var<private> g : array<i32,5,4>; // Array of 5 i32s with a stride of 4-bytes
> var<private> h : array<i32,5>; // Array of 5 i32s with an implicit stride of 4-bytes

Reply:
I don't have a good suggestion here. Enrolling into named generic parameters is certainly one way, but it's a lot to chew for this (seemingly little) problem. Leaving `<i32, 5, 4>` is just looking confusing.